packages/btcindexer/wrangler.jsonc

```diff
@@ -42,7 +42,6 @@
 
     // NOTE: for testnet-v4 it should be testnet
     "BITCOIN_NETWORK": "regtest",
-    "NBTC_DEPOSIT_ADDRESS": "bcrt1q90xm34jqm0kcpfclkdmn868rw6vcv9fzvfg6p6",
```

Comment:
- [ ] Update docs, that the nbtc addresses table has to be populated. 
- [ ] Ideally we should have a setup script as well,l
- [ ] Create `RELEASE NOTES.md` with todos for the next release.

Reply:
i will add the scirpt in a separate PR